content/04.study.md

```diff
@@ -525,9 +525,13 @@ types where the subset percentage ranged from 0.1 to 1%, significantly
 outperforming logistic regression and distance-based outlier detection methods.
 However, they did not benchmark against random forests, which tend to work
 better for imbalanced data, and their data was relatively low dimensional.
-Future work is needed to establish the utility of deep learning in cell subset
-identification, but the stunning improvements in image classification over the
-past 5 years [@tag:He2015_images] suggest transformative potential.
+
+Neural networks can also learn low-dimensional representations of single-cell gene expression data for visualization, clustering, and other tasks.
+Both scvis [@doi:10.1101/178624] and scVI [@arxiv:1709.02082] are unsupervised approaches based on VAEs.
```

Comment:
It may be worth adding one or two sentences here describing the differences between the two papers - they have different goals.

`scvis` is primarily for visualization as it compresses input data into 2 or 3 dimensions. The authors determine that their model preserves local distances better than t-sne.

`scVI` is presented more as a general modeling approach and introduces a custom zero-inflated loss into the VAE objective.

Reply:
@gwaygenomics does be27965 work better?  The sentence is long, but it was awkward to write because I can't start a new sentence with the lower-case method names.

Reply:
:+1: